packages/vscode-extension/package.json

```diff
@@ -0,0 +1,263 @@
+{
+  "name": "@trace-viewer/vscode-extension",
```

Comment:
When trying to package the extension with `vsce package` it complains that this is an invalid extension name. I think it doesn't liked the scoped naming.